lib/astc-encoder/.gitrepo

```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = https://github.com/ARM-software/astc-encoder.git
 	branch = main
-	commit = e7cb1e453968b0e16e48ef6d68fc9d1227d8a378
+	commit = 42a8f6ee01715f45edffb6773e34b8bb914a47df
 	parent = b87158d371723ba9670f5dc13e50a3e2a2799eba
```

Comment:
Change this to the correct parent of this `subrepo pull` which is 2b1c072a13e9b69ada8457665b51ed5b59ddb408.

Reply:
done in https://github.com/KhronosGroup/KTX-Software/pull/537/commits/7b67ad9c03d8c8a927e25fbefef1f619c56c8be5